In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

In [3]:
import inventorize3 as inv

In [4]:
d = 4000
c = 2500
s = 6000
h = 0.1

In [14]:
cdisc = c*0.9

In [6]:
eoq = pd.DataFrame(inv.eoq(d,s,c,h),index=[0])

In [7]:
eoq

EOQ   T_years   T_weeks
0  438.178046  0.109545  5.696315

In [8]:
eoq1 = eoq.loc[0,'EOQ']

In [11]:
tlc = (d/eoq1)*s + (eoq1/2)*h*c + (c*d)

In [12]:
tlc

10109544.511501033

In [13]:
inv.TQpractical(d,s,c,h)

{'Ts': 0.10954451150103323,
 'Tstarweeks': 5.6963145980537275,
 'Qstar': 438.1780460041329,
 'Tpractical': 4,
 'Tpracticalweeks': 4,
 'Qpractical': 307.69230769230774}

In [15]:
TLC1 = (d/500)*s + (500/2)*(h*cdisc) + d*cdisc

In [16]:
TLC1

9104250.0

Say that the Lead Time for orders to arrive is 1 month

In [17]:
t = eoq1/d

In [18]:
l = 1/12

In [19]:
l<t

True

In [20]:
reorder_point = l*d

In [22]:
reorder_point

333.3333333333333

Whenever our inventory levels are 333.33, we reorder q and q in this case is 438

When lead time = 2 months

In [31]:
l2 = 2/12

In [32]:
l2<t

False

In [33]:
l2_prime = l2 - 1*(eoq1/d)

In [34]:
l2_prime

0.05712215516563343

In [36]:
reorder_point = l2_prime*d

In [37]:
reorder_point

228.48862066253372